File: cpp/NoOfWordsInSentence.cpp
/*
 * Write a program in C++ to find the number of words present in a given sentence
 */
#include<iostream>
#include<string.h>
using namespace std;
int main ()
{
    char str[50];
    int count = 0, i; 
    cout << "Enter a sentence : ";
    gets(str);
    for (i = 0; str[i] != '\0';i++)
    {
        if (str[i] == ' ')
            count++;    
    }
    cout << "Number of words in the sentence are: " << count + 1;
    return 0;
}

File: python/NoOfWordsInSentence.py
# Write a program in Java to find the number of words present in a given sentence

# initializing string
test_string = "Hello I would like to contribute open source projects"

# printing original string
print ("The sentence is : " + test_string)

# using split()
# to count words in string
res = len(test_string.split())

# printing result
print ("The number of words in sentence are : " + str(res))
